proposals/0005-jwt-rbac.md

```diff
@@ -0,0 +1,161 @@
+# Interoperable JWT RBAC
+
+* Proposal: [MP0005-JWT_RBAC](MP0005-JWT_RBAC.md)
+* Authors: [Scott Stark](https://github.com/starksm64), [Pedro Igor Silva](https://github.com/pedroigor)
+* Status: **Awaiting review**
+
+*During the review process, add the following fields as needed:*
+
+* Decision Notes: [Discussion thread topic covering the  Rationale](https://groups.google.com/forum/#!topic/microprofile/gakCq7kSBsY), [Discussion thread topic with additional Commentary](https://groups.google.com/forum/#!topic/microprofile/l2aJGPB8Sxc)
+
+## Introduction
+
+This specification outlines a proposal for using [OpenID Connect(OIDC)](http://openid.net/connect/) based [JSON Web Tokens(JWT)](https://tools.ietf.org/html/rfc7519) for role based access control(RBAC) of microservice endpoints.
+
+Mailinglist thread: [Discussion thread topic for that proposal](https://groups.google.com/forum/#!forum/microprofile)
+
+## Motivation
+
+MicroProfile 1.1 is a baseline platform definition that optimizes Enterprise Java for a microservices architecture and delivers application portability across multiple MicroProfile runtimes. While Java EE is a very feature rich platform and is like a toolbox that can be used to to address a wide variety of application architectures, MicroProfile focuses on defining a small and a minimum set of Java EE standards that can be used to deliver applications based on a microservice architecture, are they:
```

Comment:
Based on the on going discussions, I think it would be better if each spec removed references to the MP version they're planning to associate to, since it looks like we're going to the big bang EE route and proposing umbrella specs to pull things together.